doc/source/whatsnew/v0.25.0.rst

```diff
@@ -31,6 +31,20 @@ Backwards incompatible API changes
 
 - :meth:`Timestamp.strptime` will now raise a NotImplementedError (:issue:`25016`)
 
+.. _whatsnew_0250.api_breaking.deps:
+
+Increased minimum versions for dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We have updated our minimum supported versions of dependencies (:issue:`23519`).
+If installed, we now require:
+
++-----------------+-----------------+----------+
+| Package         | Minimum Version | Required |
++=================+=================+==========+
+| pytest (dev)    | 4.0.2           |          |
++-----------------+-----------------+----------+
```

Comment:
I don't think we should put that here, that is only for contributors, not for pandas users?

Reply:
@jorisvandenbossche : This was already done in `0.24.0`, though I could go either way on this.

Reply:
This is kinda exposed to users via `pd.test`, so let's keep.